spec/services/make_an_offer_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe MakeAnOffer do
+  context 'when a conditional offer is make successfully' do
+    let(:application_choice) { create(:application_choice, provider_ucas_code: 'ABC') }
+    let(:conditions) do
+      {
+        "conditions": [
+          'Completion of subject knowledge enhancement',
+          'Completion of professional skills test',
+        ],
+      }
+    end
+    let(:make_an_offer) { MakeAnOffer.new(application_choice: application_choice, offer_conditions: conditions) }
```

Comment:
nitpick: empty line before let

Reply:
Updated in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/260/commits/d735494511a56577dab9478e780f99822a2b1bbe 👍